rtmp-client/src/main/cpp/flvmuxer/xiecc_rtmp.c

```diff
@@ -589,6 +589,9 @@ int rtmp_sender_write_video_frame(uint8_t *data,
                 fwrite(output, output_len, 1, g_file_handle);
             }
             val = RTMP_Write(rtmp, output, output_len);
+            if (val < RTMP_SUCCESS) {
```

Comment:
Does the `free(output)` need moving up to happen before this?

Reply:
I've moved it.